bridge-stp.in

```diff
@@ -212,7 +212,7 @@ case "$action" in
             # Add bridge to mstpd and configure.
             echo "Adding/configuring bridge '$bridge' ..."
             "$mstpctl" addbridge "$bridge" || continue
-            if [ -n "$config_cmd" ]; then
+            if [ -x "$config_cmd" ]; then
```

Comment:
so let's handle this as a vote here;
1) drop the check
2) keep this check unchanged
3) change to  [ -x ] || [ type -t ] as @dvlemplgk proposed

i'm a bit neutral here;  i am leaning towards 1) tho